pkg/rancher-desktop/pages/snapshots/create.vue

```diff
@@ -0,0 +1,153 @@
+<script lang="ts">
+
+import { Banner, LabeledInput, TextAreaAutoGrow } from '@rancher/components';
+import Vue from 'vue';
+
+const defaultName = () => {
+  const dateString = new Date()
+    .toISOString()
+    .slice(0, 19)
+    .replace('T', '_')
+    .replace(/-/g, '_')
+    .replace(/:/g, '_');
```

Comment:
I think that we can use dayjs to do this in a more readable way

```suggestion  
const dateString = dayjs().format('YYYY-MM-DD_HH_mm_ss');
```